ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/CreateStreamCommand.java

```diff
@@ -45,8 +46,10 @@ public DDLCommandResult run(MetaStore metaStore) {
         sqlExpression,
         sourceName,
         schema,
-        (keyColumnName.length() == 0) ? null : schema.field(keyColumnName),
-        (timestampColumnName.length() == 0) ? null : schema.field(timestampColumnName),
+        (keyColumnName.length() == 0) ?
+            null : SchemaUtil.getFieldByName(schema, keyColumnName).orElse(null),
```

Comment:
nit: might be better formatting if the `: SchemaUtil...` was on its own line. Same below